lib/internal/encoding.js

```diff
@@ -443,6 +446,10 @@ function makeTextDecoderICU() {
         return decodeUTF8(input, this[kIgnoreBOM], this[kFatal]);
       }
 
+      if (this[kLatin1FastPath]) {
```

Comment:
After Line 443, we need this, since we don't support options.stream on the fast path.

```
this[kLatin1FastPath] &&= !(options?.stream)
```

Reply:
I will apply it as you say, thank you.